lbz/misc.py

```diff
@@ -16,7 +17,7 @@ class NestedDict(dict):
     Endless nested dict.
     """
 
-    def __getitem__(self, key):
+    def __getitem__(self, key: str) -> Union[Any, dict]:
```

Comment:
It should return just `-> Any` as `dict` is one of `Any` 😉  This class is also very generic, so returning `Any` here makes perfect sense.